vhdbuilder/packer/generate-vhd-publishing-info.sh

```diff
@@ -51,6 +51,7 @@ else
         "os_name" : "$OS_NAME",
         "sku_name" : "$sku_name",
         "offer_name" : "$OFFER_NAME"
+        "hyperv_generation": "${HYPERV_GENERATION}"
```

Comment:
I think you need to add it to required_env_vars

Reply:
sure let me add it